web-gui/src/app/components/courses/detail-course/update-course-dialog/update-course-dialog.component.ts

```diff
@@ -26,19 +25,18 @@ export class UpdateCourseDialogComponent implements OnInit {
   coursenameMinLength: number = 5;
   courseNameMaxLength: number = 100;
   courseDescriptionMaxLength: number = 8000;
-  coursename = new FormControl('', [Validators.required, Validators.minLength(this.coursenameMinLength), Validators.maxLength(this.courseNameMaxLength)])
-  courseDescription = new FormControl('', [Validators.maxLength(this.courseDescriptionMaxLength)]);
+  // tslint:disable-next-line:max-line-length
```

Comment:
Is it not possible to make a line break instead?

Reply:
No because 8000 character are the db limit.

Reply:
Ah sorry, You cannot see the line that I mean.
The line after `// tslint:disable-next-line:max-line-length` exceeds the max-line length.
Can we make a line break there and remove`// tslint:disable-next-line:max-line-length`?

Reply:
Ah okay, than of course yes :) I will changed that!